app/main/forms/auth_forms.py

```diff
@@ -28,6 +28,7 @@
 PASSWORD_MISMATCH_ERROR_MESSAGE = "The passwords you entered do not match"
 NEW_PASSWORD_EMPTY_ERROR_MESSAGE = "Enter a new password"
 NEW_PASSWORD_CONFIRM_EMPTY_ERROR_MESSAGE = "Confirm your new password"
+PASSWORD_CHANGE_AUTH_ERROR_MESSAGE = "Make sure you’ve entered the right password."
```

Comment:
Would this work?: Enter your old password

Reply:
Sounds good to me! Will edit, thanks Nora 👍